test/lib/ansible_test/_data/sanity/validate-modules/validate_modules/main.py

```diff
@@ -2212,7 +2211,7 @@ def validate(self):
 
```

Comment:
A couple of lines up (line 2206) there's still a `PS_DOC_BLACKLIST` left (which causes CI failure).